<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GraphicsDataNode</class>
 <widget class="QWidget" name="GraphicsDataNode">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>250</width>
    <height>122</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">

background: rgba(255, 255, 255, 0);
    color: #BBBBBB;
    font-family: &quot;Segoe UI&quot;;

</string>
  </property>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>50</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">
QComboBox {border-radius:10px;background:rgb(97, 97, 97);color:#ffffff;
padding-left:30px;
}  

QComboBox QAbstractItemView {background:rgb(189, 189, 189);color:#ffffff;} 
 

QComboBox::drop-down{
width: 20px;
border-left: 1px solid #ffffff; 
}    

QComboBox::down-arrow {image: url(:/img/img/xia.png);height:40px;width:20px;}   

QListView::item{padding-left:30px;background:rgb(152, 152, 152);color:#ffffff;}  

QListView::item:selected{ 
color:#DCDCDC;
background:rgb(94,167,255);
}   

QListView::item:hover{ 
color:#DCDCDC;
background:rgb(94,167,255);
}</string>
   </property>
   <item>
    <property name="text">
     <string>String</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>float</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>double</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>int</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>long long</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="nodeBackground">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>231</width>
     <height>121</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">

background: #222222;
    color: #BBBBBB;
    font-family: &quot;Segoe UI&quot;;
border-radius:50%;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>90</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>9</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border:0px;
border-radius:10px;</string>
   </property>
   <property name="text">
    <string>Hello World!</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>10</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>9</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border:3px solid rgb(126, 126, 126);
border-radius:10px;</string>
   </property>
   <property name="text">
    <string> 数据节点</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>50</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>8</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>out</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>8</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>in</string>
   </property>
  </widget>
  <widget class="Port" name="inputPort">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
  </widget>
  <widget class="Port" name="outputPort">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>50</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
  </widget>
  <zorder>nodeBackground</zorder>
  <zorder>comboBox</zorder>
  <zorder>lineEdit</zorder>
  <zorder>lineEdit_2</zorder>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
  <zorder>inputPort</zorder>
  <zorder>outputPort</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Port</class>
   <extends>QWidget</extends>
   <header>port.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
